TaskMonitorPlus/__init__.py

```diff
@@ -127,6 +142,7 @@ def CheckWindow(self, hwnd):
         return processInfo
 
     def MyWndProc(self, dummyHwnd, dummyMesg, wParam, lParam):
+        eg.actionThread.Call(self.CheckPlugins)
```

Comment:
now since we have to do all of the actions to the plugins in the action thread. because if we do not we will get errors if running in debug mode. so we tell the action thread to run the CheckPlugins method whenever it manages to get around to it.